src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,318 @@
+                          Died
+
+                          <SearchLink
+                            params={sortParams('died')}
+                          >
+                            <span className="icon" aria-label="died">
+                              {sortArrows('died')}
+                            </span>
+                          </SearchLink>
+                        </span>
+                      </th>
+                      <th>Mother</th>
+                      <th>Father</th>
+                    </tr>
+                  </thead>
+
+                  <tbody>
+                    {necessaryPeople.map(people => {
+                      const isActive = slug === people.slug;
+                      const trClassName = isActive
+                        ? 'has-background-warning'
+                        : '';
+
+                      return (
+                        <tr
+                          data-cy="person"
+                          key={people.slug}
+                          className={trClassName}
+                        >
+                          <td>
+                            <NavLink
+                              to={`${people.slug}`}
+                              className={classNames(
+                                {
+                                  'has-text-danger': people.sex === 'f',
+                                },
+                              )}
+                            >
+                              {people.name}
+                            </NavLink>
+                          </td>
+
+                          <td>{people.sex}</td>
+                          <td>{people.born}</td>
+                          <td>{people.died}</td>
+                          {necessaryPeople.find(mother => {
+                            return mother.name === people.motherName;
+                          }) ? (
+                              peoples.map(mother => {
+                                if (mother.name === people.motherName) {
+                                  return (
+                                    <td>
+                                      <NavLink
+                                        to={`${mother.slug}`}
+                                        className="has-text-danger"
+                                      >
+                                        {mother.name}
+                                      </NavLink>
+                                    </td>
+                                  );
+                                }
+
+                                return null;
+                              })
+                            ) : (
+                              <td>
+                                {
+                                  people.motherName ? people.motherName
+                                    : '-'
+                                }
+                              </td>
+                            )}
+
```

Comment:
```suggestion
                                
                                  return (
                                    <td>
                                    {mother ? (
                                      <NavLink
                                        to={`${mother.slug}`}
                                        className="has-text-danger"
                                      >
                                        {mother.name}
                                      </NavLink>                                      
                                      ) : people.motherName || "-"}
                                    </td>
                                  );
                             
```

Reply:
I don't really understand how to implement it